<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>857</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CG</string>
  </property>
  <widget class="QWidget" name="mainWindowWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1" columnstretch="1,1,0">
    <item row="0" column="0" rowspan="3">
     <widget class="QGroupBox" name="menuGroupBox">
      <property name="title">
       <string>Menu</string>
      </property>
      <layout class="QVBoxLayout" name="menuLayout">
       <item>
        <widget class="QPushButton" name="addGraphicalObjectButton">
         <property name="text">
          <string>Add Graphical Object</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="listView"/>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Window</string>
         </property>
         <layout class="QGridLayout" name="toolsLayout" rowstretch="0,0,0,0,0" columnstretch="1,1,1,1">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item row="4" column="2">
           <widget class="QPushButton" name="moveDownPushButton">
            <property name="text">
             <string>↓</string>
            </property>
            <property name="shortcut">
             <string>Down</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLineEdit" name="zoomLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QLineEdit" name="positionLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2" colspan="2">
           <widget class="QPushButton" name="moreZoomPushButton">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="standartZoomPushButton">
            <property name="text">
             <string>100%</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="zoomLabel">
            <property name="text">
             <string>Zoom</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QPushButton" name="centerPushButton">
            <property name="text">
             <string>0,0</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QPushButton" name="moveRightPushButton">
            <property name="text">
             <string>→</string>
            </property>
            <property name="shortcut">
             <string>Right</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="lessZoomPushButton">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="moveUpPushButton">
            <property name="text">
             <string>↑</string>
            </property>
            <property name="shortcut">
             <string>Up</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="moveLeftPushButton">
            <property name="text">
             <string>←</string>
            </property>
            <property name="shortcut">
             <string>Left</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="positionLabel">
            <property name="text">
             <string>Posição</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="rotateLabel">
            <property name="text">
             <string>Rotação</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="counterClockwiseRotatePushButton">
            <property name="font">
             <font>
              <family>Courier New</family>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../application.qrc">
              <normaloff>:/icons/resources/rotateRight.png</normaloff>:/icons/resources/rotateRight.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="clockwiseRotatePushButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../application.qrc">
              <normaloff>:/icons/resources/rotateLeft.png</normaloff>:/icons/resources/rotateLeft.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="rotateLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QGroupBox" name="consoleGroupBox">
      <property name="title">
       <string>Console</string>
      </property>
      <layout class="QVBoxLayout" name="consoleLayout">
       <item>
        <widget class="QTextBrowser" name="consoleTextBrowser"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2" colspan="2">
     <widget class="QGroupBox" name="canvasGroupBox">
      <property name="title">
       <string>Canvas</string>
      </property>
      <layout class="QVBoxLayout" name="canvasLayout">
       <item>
        <widget class="GViewport" name="viewport" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>500</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>500</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>500</width>
           <height>500</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>992</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GViewport</class>
   <extends>QWidget</extends>
   <header>model/gviewport.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../application.qrc"/>
 </resources>
 <connections/>
</ui>
